.github/workflows/typescript.yml

```diff
@@ -47,10 +47,6 @@ jobs:
       - name: Install dependencies
         run: yarn install
 
-      - name: Install uv
```

Comment:
can we add `./.github/actions/untracked` in this workflow to detect untracked files?